src/models/flags.ts

```diff
@@ -68,32 +68,37 @@ export const categoriesAndFlags: Record<string, Record<string, string[]>> = {
   },
 };
 
-export const flatFlags: Array<Flag> = [];
```

Comment:
This looks like we're exporting an empty array but we're actually setting up an array that is mutated later.
This is a lot easier to understand if we export *after* we've constructed the array.